pkgs/kubernetes/kubectl/registry.yaml

```diff
@@ -23,14 +20,8 @@ packages:
           file_format: raw
     version_constraint: semver(">= 1.23.0")
     version_overrides:
-      - version_constraint: semver(">= 1.21.0")
-        supported_envs:
-          - darwin
-          - linux
-          - amd64
-      - version_constraint: "true"
-        rosetta2: true
+      - version_constraint: semver("< 1.23.0")
```

Comment:
version_overrides has been changed. 🤔
I'm not sure why this change is needed.

Reply:
Looking at the Kubernetes static binary download site, windows/arm64 binaries only exist since 1.23.0.
https://www.downloadkubernetes.com/

Also, since 1.22.0 includes Windows binaries, we thought it would be a good idea to update `supported_envs`.

Reply:
Sorry, I thought about it and the only change I made to version_overrides was to add Windows. I'll revert it back to the original.